draft-ietf-quic-transport.md

```diff
@@ -2123,7 +2123,22 @@ unable to due to connection-level flow control (see {{blocking}}). BLOCKED
 frames can be used as input to tuning of flow control algorithms (see
 {{fc-credit}}).
 
-The BLOCKED frame does not contain a payload.
+The BLOCKED frame is as follows:
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                         Offset (i)                         ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+
+The BLOCKED frame contains a single field.
+
+Offset:
+
+: A variable-length integer indicating the current offset of data sent on the
```

Comment:
current is ambiguous.  I presume that the intent is to signal the offset at which the blocking occurred.